src/resdk/tables/microarray.py

```diff
@@ -0,0 +1,108 @@
+""".. Ignore pydocstyle D400.
+
+========
```

Comment:
You should also add some lines to the topic documentation.

Reply:
Good point, thanks!